eduVPN/ConnectionService.swift

```diff
@@ -282,9 +282,16 @@ class ConnectionService: NSObject {
         if developerMode {
             scriptOptions.append("-l" /* ARG_EXTRA_LOGGING */)
         }
-       
+        
         self.configURL = configURL
-        helper.startOpenVPN(at: openvpnURL, withConfig: configURL, upScript: upScript, downScript: downScript, leasewatchPlist: leasewatchPlist, leasewatchScript: leasewatchScript, scriptOptions: scriptOptions) { (success) in
+        helper.startOpenVPN(at: openvpnURL, withConfig: configURL, upScript: upScript, downScript: downScript, leasewatchPlist: leasewatchPlist, leasewatchScript: leasewatchScript, scriptOptions: scriptOptions) { (status) in
+            
+            var data:Array? = status
```

Comment:
Create a class in objective C containing the values you need to pass along for example success  status, disallowed commands, etc.